frontend/src/metabase/css/dashboard.module.css

```diff
@@ -205,10 +163,6 @@
   transition: all 0.15s linear;
 }
 
-:global(.Dash--editing .Card-title) {
```

Comment:
The previous comment applies. Please double check if this rule is still used or not.